appinventor/components/src/com/rit/appinventor/components/runtime/ros_bridge.java

```diff
@@ -7,21 +7,25 @@
 import com.google.appinventor.components.annotations.SimpleFunction;
 import com.google.appinventor.components.annotations.SimpleObject;
 import com.google.appinventor.components.common.ComponentCategory;
-import com.google.appinventor.components.common.YaVersion;
 import com.google.appinventor.components.runtime.AndroidNonvisibleComponent;
 import com.google.appinventor.components.runtime.Deleteable;
 import com.google.appinventor.components.runtime.OnResumeListener;
 import com.google.appinventor.components.runtime.OnStopListener;
+import com.google.appinventor.components.runtime.ComponentContainer;
```

Comment:
Do we have to import the libraries or does it know based off of the build.xml file?

Reply:
We still have to import the new libraries. But for IntelliJ to work we need to add them to the build path in the project structure settings. A.k.a the black magic that George did to my project last tuesday. We can work on it again next meeting

Reply:
Sounds good